File: crabzilla-core/src/main/java/io/github/crabzilla/core/entity/SnapshotData.java
package io.github.crabzilla.core.entity;

import io.github.crabzilla.core.DomainEvent;
import lombok.Value;

import java.io.Serializable;
import java.util.List;

@Value
public class SnapshotData implements Serializable {

  private Version version;
  private List<DomainEvent> events;

}

File: crabzilla-core/src/main/java/io/github/crabzilla/core/entity/SnapshotData.kt
package io.github.crabzilla.core.entity

import io.github.crabzilla.core.DomainEvent
import io.github.crabzilla.core.Version
import java.io.Serializable

class SnapshotData(val version: Version, val events: List<DomainEvent>) : Serializable
